wpeverest/everest-forms
# Tweak - Multiple CSS class on form and its fields option

### All Submissions:






### Changes proposed in this Pull Request:



Closes #118 

### How to test the changes in this Pull Request:

1. While adding form block set additional CSS it should validate multiple CSS classes.
2. Goto form General settings, set form CSS classes, it should validate and work great.
2. Select a form and drag a field, goto its advanced option and set form classes, it should work great.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Multiple CSS classes option in the fields.
